src/__tests__/notifications.test.ts

```diff
@@ -0,0 +1,334 @@
+    });
+
+    await expect(
+      notifications.postVariations("notification1", {
+        blocks: [
+          {
+            id: "block_00d0bcb0-aba1-443f-a8dd-daac505500fe",
+            type: "text",
+            locales: {
+              fr_FR: "french text block"
+            }
+          },
+          {
+            id: "block_41f7d5c0-bfeb-49fb-b3be-fee7c84f4d45",
+            type: "action",
+            locales: {
+              fr_FR: "french action block"
+            }
+          }
+        ],
+        channels: [
+          {
+            id: "channel_79d25574-83be-4da1-a5c3-3d4e2ab5f154",
+            locales: {
+              fr_FR: {
+                email: {
+                  subject: "French hey!"
+                }
+              }
+            }
+          },
+          {
+            id: "channel_76051a88-9fd6-4fde-9d9f-4f77f83d0fea",
+            locales: {
+              fr_FR: {
+                push: {
+                  title: "French tk"
+                }
+              }
+            }
+          }
+        ]
+      })
+    ).resolves.toBeUndefined();
+  });
+
+  test(".post notification draft variations", async () => {
+    const { notifications } = CourierClient({
+      authorizationToken: "AUTH_TOKEN"
+    });
+
+    await expect(
+      notifications.postDraftVariations("notification1", {
+        blocks: [
+          {
+            id: "block_00d0bcb0-aba1-443f-a8dd-daac505500fe",
+            type: "text",
+            locales: {
+              fr_FR: "french text block"
+            }
+          },
+          {
+            id: "block_41f7d5c0-bfeb-49fb-b3be-fee7c84f4d45",
+            type: "action",
+            locales: {
+              fr_FR: "french action block"
+            }
+          }
+        ],
+        channels: [
+          {
+            id: "channel_79d25574-83be-4da1-a5c3-3d4e2ab5f154",
+            locales: {
+              fr_FR: {
+                email: {
+                  subject: "French hey!"
+                }
+              }
+            }
+          },
+          {
+            id: "channel_76051a88-9fd6-4fde-9d9f-4f77f83d0fea",
+            locales: {
+              fr_FR: {
+                push: {
+                  title: "French tk"
+                }
+              }
+            }
+          }
+        ]
+      })
+    ).resolves.toBeUndefined();
+  });
+
+  test(".get notification submission checks", async () => {
+    const { notifications } = CourierClient({
+      authorizationToken: "AUTH_TOKEN"
+    });
+
+    await expect(
+      notifications.getSubmissionChecks("notification1", "submission1")
+    ).resolves.toMatchObject(mockGetNotificationSubmissionChecksResponse);
+  });
+
+  test(".put notification submission checks", async () => {
+    const { notifications } = CourierClient({
+      authorizationToken: "AUTH_TOKEN"
+    });
+
+    await expect(
+      notifications.putSubmissionChecks("notification1", "submission1", {
```

Comment:
what is the use of `submission1`?

Reply:
a check is associated with a submissionId (in this case submission1)